aws-sns-topicinlinepolicy/src/main/java/software/amazon/sns/topicinlinepolicy/DeleteHandler.java

```diff
@@ -10,7 +10,7 @@
 import software.amazon.cloudformation.proxy.HandlerErrorCode;
 
 public class DeleteHandler extends BaseHandlerStd {
-    private software.amazon.cloudformation.proxy.Logger logger;
+    private software.amazon.cloudformation.proxy.Logger Log;
```

Comment:
Why do you need to change logger to Log? Is this just a variable name change?

Reply:
No idea. Because spotbugs set it as an error..